src/nixpacks/images.rs

```diff
@@ -1,3 +1,3 @@
-pub static DEFAULT_BASE_IMAGE: &str = "ghcr.io/railwayapp/nixpacks:debian-1654798583";
+pub static DEFAULT_BASE_IMAGE: &str = "ghcr.io/railwayapp/nixpacks:debian-1654800960";
```

Comment:
This also seems unrelated. A rebase would remove this.

Reply:
How will a rebase remove this?

Reply:
You're not changing the image in this PR, so a rebase would remove this change

Reply:
I am updating the image to `debian-1654800960`(canvas requires that). Currently nixpacks uses `debian-1654798583` and that's not the latest release.

Reply:
I thought I saw a PR get merged about updating this?

Reply:
You saw that right but in that same PR jr made some changes to the base Dockerfile.